src/Components/Search/NewSearch/constants.ts

```diff
@@ -65,3 +67,19 @@ export const PILLS: PillType[] = [
   SHOW_PILL,
   GALLERY_PILL,
 ]
+
+interface SearchEntityMap {
+  [key: string]: SearchEntity
+}
```

Comment:
JFYI we can use [Record](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) from TS. Something like this

```javascript
export const ELASTIC_PILL_KEY_TO_SEARCH_ENTITY: Record<string, SearchEntity> = {...}
```